### DISCUSSION TITLE
Where is the MSP430 GCC?

### DISCUSSION THREAD

**Discussion by tunguskar** [compiler-explorer/compiler-explorer#5237] [Q&A]
Hi,

I used the MSP430 Compiler a lot, but cant find it anymore. Was it removed from compiler-explorer?

Kind Regards

Christian

**Comment by partouf** [comment]
They're definitely there

Under C++ there are these:

![image](https://github.com/compiler-explorer/compiler-explorer/assets/442630/4cc630b8-a023-4e63-ba26-c2ecd030047e)

There's more under C

**Reply by tunguskar** [reply]
Thanks for the reply @partouf. when I put `msp430`in the search line on my laptop at work I did not get the msp430 compiler but on my laptop at home it works. Strange

<img width="249" alt="image" src="https://github.com/compiler-explorer/compiler-explorer/assets/2837624/4d923d7c-aa8c-40e9-b49a-9dd4bb10d077">

**Comment by mattgodbolt** [comment]
@tunguskar is it possible you have "C" instead of "C++" as the language on your work laptop? Or something like that?

**Reply by tunguskar** [reply]
Hi Matt, 

Did some tests. on mein Laptop at work (windows with edge browser) it looks like this:

![image](https://github.com/compiler-explorer/compiler-explorer/assets/2837624/36aaf32f-8194-494b-a9d9-2b06a92d7778)
